src/main/scala/uk/gov/hmrc/hmrcfrontend/views/viewmodels/listwithactions/ListWithActions.scala

```diff
@@ -24,7 +24,9 @@ case class ListWithActions(
   items: Seq[ListWithActionsItem] = Nil,
   classes: String = "",
   attributes: Map[String, String] = Map.empty
-)
+) {
+  val anyRowHasActions: Boolean = items.exists(_.actions.nonEmpty)
```

Comment:
Might be worth a chat about this. Historically, we have tended to put this sort of logic into the Twirl template, so that logic isn't split between viewmodel and view, but in one or the other. I would prefer if we could stick to that convention, as the rest of the display logic is in the `HmrcListWithActions.scala.html` template.